doc/manual.rst

```diff
@@ -2975,6 +2975,21 @@ Even some code that has side effects is permitted in a static block:
   static:
     echo "echo at compile time"
 
+When `static` is used like a procedure call with a proc call as an argument,
+that proc call gets executed at compile-time.
```

Comment:
```suggestion
`static` can also be used like a routine.
```
Your version implies that the argument has to be a proc call, but it can be any expression.